optuna/study/study.py

```diff
@@ -1252,8 +1252,8 @@ def objective(trial):
         for d in directions
     ):
         raise ValueError(
-            "Please set either 'minimize' or 'maximize' to direction. You can also set the "
-            "corresponding `StudyDirection` member."
+            "'direction' must be 'minimize' or 'maximize'."
+            " For multiple objectives, use 'directions'."
```

Comment:
The problem here happens when users pass `directions` instead of `direction`.
This statement is triggered when `directions` is passed anyways, so it would be better to say that users used `directions` in a wrong way instead of just saying `use directions`.
 
```suggestion
            f"`directions` must be a list of `minimize` or `maximize`, but got {directions}. "
            "For single-objective optimization, please use `direction` instead of `directions`."
```